README.md

```diff
@@ -1 +1,14 @@
-# cocoapod-specs
\ No newline at end of file
+# The Bitmovin CocoaPods Repo
+
+This repository contains all public CocoaPods specifications provided by [Bitmovin](https://bitmovin.com).
```

Comment:
> all

I'd probably omit the `all` as analytics isn't in here (yet?), and also none of the player integrations (like Conviva) are in here (yet?).

Reply:
Good point. Eventually all pods should be available through this repo 🙂

Reply:
4168ae8

Reply:
That's the dream 🤩  but might take some more time 😄